.travis.yml

```diff
@@ -1,13 +1,27 @@
 language: rust
 
+rust:
+  - stable
+  - beta
```

Comment:
I must admit, I'm not very sure it's useful for us to test on `beta`. I know it doesn't *hurt*, but I feel a bit bad for using electricity for something that's probably never going to show an error.

Reply:
Seems like a good way to eliminate a lot of builds as well

Reply:
done.